app/init.php

```diff
@@ -475,6 +476,20 @@ function (mixed $value, Document $document, Database $database) {
     }
 );
 
+Database::addFilter(
+    'subQueryDevKeys',
```

Comment:
I dont see changes to project response model.

Reply:
updated.